src/frontend/Sider/Behandling/Venstremeny/Oppsummering/Utdanning.tsx

```diff
@@ -32,9 +32,9 @@ const Utdanning: React.FC<{ faktaUtdanning: FaktaUtdanning }> = ({ faktaUtdannin
                     </BodyShort>
                 </InfoSeksjon>
             )}
-            {mottarUtstyrsstipend && (
-                <InfoSeksjon label="Utstyrstipend" ikon={<BankNoteIcon />}>
-                    <BodyShort size="small">{jaNeiTilTekst[mottarUtstyrsstipend]}</BodyShort>
+            {erLærlingEllerLiknende && (
+                <InfoSeksjon label="Er lærling eller liknende" ikon={<HatSchoolIcon />}>
```

Comment:
Mulig det burde stå 'Er lærling, lærekandidat, praksisbrevkandidat eller kandidat for fagbrev på jobb?' ?

Reply:
Går sikkert å høre med Marie om det, tenker det er greit 👍